docs/deserialization.rst

```diff
@@ -0,0 +1,64 @@
+===============
+Deserialization
+===============
+Deserialization is similar to serialization. Deserialization allows native Python datatypes to be converted to Python objects.
+
+Defining our deserailizer
+========================
+
+Lets consider a similar example taht we used in serialization example. Lets assume we have a comment object and the comment object has a user attached to it, Now lets define a deserializer class::
+
+    from pyserializer.serializers import Serializer
+    from pyserializer import fields
+
+
+    class UserDeserializer(Serializer):
+        email = fields.CharField()
+        username = fields.CharField()
+
+        class Meta:
+            fields = (
+                'email',
+                'username'
+            )
+
+        def __repr__(self):
+            return '<User(%r)>' % (self.username)
+
+
+    class CommentDeserializer(Serializer):
+        user = UserDeserializer()
+        content = fields.CharField()
+        created_date = fields.DateField(format='%Y-%m-%d')
+        created_time = fields.DateTimeField(format='%Y-%m-%dT%H:%M:%SZ')
+
+        class Meta:
```

Comment:
I'm not clear on the distinction between the fields defined as properties on the class, and fields defined in `Meta`.

From looking at the `Serializer` code, it looks like you only need to define `Meta` if you want a different metaclass for your serializer options, but I don't see an example of doing that. If I'm understanding correctly, the `class Meta` blocks could be omitted from these examples and they would still work.

So assuming I'm not missing something, I think the docs would be improved by: removing the Meta definitions from these examples, and adding a new example that shows when you *would* want to override Meta.